releasenotes/notes/fix-cluster-name-tag-normalization-cd0b7cac8a1a761f.yaml

```diff
@@ -0,0 +1,18 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Introduce a new option `enabled_rfc1123_compliant_cluster_name_tag`
+    that can be use to enfore `kube_cluster_name` tag value to be
+    RFC1123 Compliant cluster name.
+fixes:
+  - |
+    Fix in `kubernetes_state_core` check the host alias name
+    create to use a normalize (RFC1123 Compliant) cluster name.
```

Comment:
```suggestion
    Fix the `kubernetes_state_core` check so that the host alias name
    creation uses a normalized (RFC1123 compliant) cluster name.
```

Reply:
Let me know if the proposed changes here introduce any inaccuracies to the content.